docs/glossary.md

```diff
@@ -643,7 +643,19 @@ Content Delivery Network
 CDN
     A Content Delivery Network (CDN) is a network of servers located in various geographic regions that work together to deliver web content to users quickly and efficiently.
 
+
+unique identifier
+UID
+   UID is an acronym meaning "unique identifier".
+   A UID is an identifier that is guaranteed to be unique among all identifiers used for those objects and for a specific purpose.
+
+integer identifier
+intid
+    In Plone, an integer identifier, or intid, is used to uniquely identify content objects within a Plone site.
+    Each content item in a Plone site is given a unique intid, which the system uses internally to reference content, keep track of link-integrity, link translations, and other related purposes.
+
 WSL
 Windows Subsystem for Linux
     The [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) lets developers install a Linux distribution (such as Ubuntu, OpenSUSE, Kali, Debian, or Arch Linux) and use Linux applications, utilities, and Bash command-line tools directly on Windows, unmodified, without the overhead of a traditional virtual machine or dualboot setup.
+
```

Comment:
Let's keep this together.

```suggestion
```

You may add a blank line at the very end, if GitHub complains with a red circle with a slash.